pkg/providers/korifi/models/korifi_types.go

```diff
@@ -0,0 +1,235 @@
+package models
+
+// From https://github.com/cloudfoundry/korifi/blob/main/api/presenter/info.go#L10
+// ------------------------------------------------
+
+type SpaceResponse struct {
```

Comment:
why do you need to copy this here? can't we import the library?

Reply:
Done